src/print-env.spec.js

```diff
@@ -0,0 +1,77 @@
+'use strict'
+
+/* eslint-env mocha */
+const printEnv = require('.')
+const clone = x => JSON.parse(JSON.stringify(x))
+const snapshot = require('snap-shot-it')
+const execa = require('execa')
+const { join } = require('path')
+
+describe('@bahmutov/print-env', () => {
+  let env
+
+  beforeEach(() => {
+    env = clone(process.env)
```

Comment:
we can either use `extendEnv` (see below) or https://github.com/bahmutov/mocked-env to mock the environment object

Reply:
Node processes can't permanently alter the environment, so I believe we can safely overwrite `process.env` without having to restore it later.